apps/backend/src/lib/ai/checkpointer.ts

```diff
@@ -0,0 +1,41 @@
+import { PostgresSaver } from "@langchain/langgraph-checkpoint-postgres"
+import type { Pool } from "pg"
+
+import { logger } from "../logger"
+
+const LANGGRAPH_SCHEMA = "langgraph"
+
+let checkpointer: PostgresSaver | null = null
+
+/**
+ * Initialize the LangGraph PostgreSQL checkpointer.
+ *
+ * This creates the necessary tables in the `langgraph` schema on first run.
+ * Must be called once at server startup before any graph invocations.
+ */
+export async function initCheckpointer(pool: Pool): Promise<PostgresSaver> {
```

Comment:
Can we rename this file to `postgresql-checkpointer.ts` instead as that's what it is?